src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/EnvelopeProcessor.java

```diff
@@ -147,6 +151,29 @@ public static void assertEnvelopeHasPdfs(InputEnvelope envelope, List<Pdf> pdfs)
         }
     }
 
+    /**
+     * Assert envelope contains at least one scannable item
+     * with valid document type and ocr data for new application.
```

Comment:
No mention of the fact that this is only checked for SSCS jurisdiction.

This function seems very specific the way it is right now.
Perhaps it could take parameters?

Reply:
Updated javadoc to include the conditions for jurisdiction and document_type